assets/helpers/page/page.js

```diff
@@ -49,6 +52,15 @@ function analyticsInitialisation(participations: Participations): void {
   ga.setDimension('experience', abTest.getVariantsAsString(participations));
   ga.trackPageview();
 
+  const dimensions:Dimensions = {
+    campaignCodeBusinessUnit: getQueryParameter('CMP_BUNIT') || undefined,
+    campaignCodeTeam: getQueryParameter('CMP_TU') || undefined,
+    experience: abTest.getVariantsAsString(participations),
+  };
+
+  googleTagManager.init();
```

Comment:
Have you tested this with the do not track header set?

It would be good if we could avoid calling it when it is.

Reply:
Good call. I will do a little research to read the DNT from this function.

Reply:
There is an experimental way of checking the DoNotTrack flag from client side: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack

Reply:
Since the init function only sets up a `window.googleTagManagerDataLayer`, in the case of do not track flag is set, this will still be working. Nevertheless, I will add the extra `navigator` check.